app/views/admin/decks/edit.html.erb

```diff
@@ -15,4 +15,14 @@
 <div>
   <%= button_to "Delete Card", admin_deck_path, :method => "delete" %>
 </div>
+<hr>
+<h1>Cards In Deck</h1>
+
+<table>
+<% @memberships.each do |membership| %>
```

Comment:
After we make those changes in `admin/decks_controller.rb`, we can instead do `@cards.each` and display each card that way.